### PR TITLE
Arachne 1729

### PR DESCRIPTION
**Pull request by mrAlexandr**

### CODE REVIEW

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  return <span>{!!persons ? persons : 0} person{persons !== 1 ? "s" : ""}</span>;
+}
+
+function CohortCharacterization(props) {
+  const {
+    resultInfo,
+  } = props;
+
+  return (<span>{`${resultInfo.reports} report${resultInfo.reports > 1 || resultInfo.reports === 0 ? "s" : ""}`}</span>);
```
Comment (johnSamilin) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: !== 1

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  const proportion = get(resultInfo, "PROPORTION");
+  const tooltipString = `person${personCount > 1 ? "s" : ""}: ${personCount} 
+  time at risk:${timeAtRisk} 
+  case${cases > 1 ? "s" : ""}:${cases} 
+  rate: ${rate} 
+  proportion: ${proportion}`;
+
+  return <div {...classes({
+      extra: tooltipClass().className,
+      // modifiers: {'line'}
```
Comment (johnSamilin) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: unneccessary comment

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      element: 'line'})} aria-label={tooltipString} data-tootik-conf="left">
```
Comment (johnSamilin) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: data-tootik-conf="left multiline"

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/style.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      @include material-icon();
+      font-size: 1.6rem;
+    }
+
+    &__line{
+      display: flex;
+      align-items: center;
+    }
+
+    &__tooltip {
```
Comment (johnSamilin) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/style.scss]: not exist

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+function Incidence(props) {
+
+  const {
+    resultInfo,
+  } = props;
+
+  const classes = new BEMHelper('incedence-label');
+  const tooltipClass = new BEMHelper('tooltip');
+
+  const personCount = get(resultInfo, "PERSON_COUNT");
```
Comment (johnSamilin) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: prefer single quotes

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+function Cohort(props) {
+
+  const {
+    resultInfo,
+  } = props;
+
+  const persons = get(resultInfo, 'persons');
+
+  return <span>{!!persons ? persons : 0} person{persons !== 1 ? 's' : ''}</span>;
```
Comment (pavgra) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: Create or find some util helper, like "pluralize"

Comment (pavgra) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: Why not "persons || 0" ?

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+ * Created: December 27, 2016
```
Comment (agackovka) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/ListSubmissions/Results/presenter.js]: Where did you get this year?)

**Pull request merged**